cpp/CMakeLists.txt

```diff
@@ -902,6 +897,11 @@ target_compile_definitions(cudf PUBLIC "SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LIBCU
 # Enable remote IO through KvikIO
 target_compile_definitions(cudf PRIVATE $<$<BOOL:${CUDF_KVIKIO_REMOTE_IO}>:CUDF_KVIKIO_REMOTE_IO>)
 
+# Enable cuFile support
+if(TARGET CUDA::cuFile)
+  target_compile_definitions(cudf PRIVATE CUFILE_FOUND)
```

Comment:
Could we rename this to `CUDF_CUFILE_FOUND`? We have the name `KVIKIO_CUFILE_FOUND` in kvikio.

Reply:
I've renamed it locally, will push once CI passes.